### ISSUE TITLE
Contrast ratio on Privacy and Terms links does not meet WCAG 2.0 AA SC 1.4.3 text contrast of 4.5:1

### ISSUE THREAD

**Issue by amyleesalesforce** [google/recaptcha#457]
The white-on-blue text of the Privacy and Terms hover does not meet sufficient contrast for WCAG 2.0 AA. The white is currently #FFFFFF on blue background #4A90E2, which is 3.3:1 contrast.

Please darken the blue background, e.g. #2275D3. Thank you!

Related:
* https://github.com/google/recaptcha/issues/443
* https://github.com/google/recaptcha/issues/430

![3 2 - 3](https://user-images.githubusercontent.com/7945524/116326015-5510a380-a778-11eb-96b7-f1d31b3c8668.png)
